site2/doc-guides/contribution.md

```diff
@@ -0,0 +1,219 @@
+
+Files in [pulsar/site2 (master branch)](https://github.com/apache/pulsar/tree/master/site2) are synced to [pulsar-site/website-next (main branch)](https://github.com/apache/pulsar-site/tree/main/site2/website-next) every 6 hours. You can check the sync status and progress at [pulsar-site Actions](https://github.com/apache/pulsar-site/actions/workflows/ci-pulsar-website-docs-sync.yaml).
+
+> **Summary**
+>
+> * pulsar/site2/**website** = pulsar-site site2/**website-next** 
+> 
+> * pulsar/site2/**docs** = pulsar-site/site2/**website-next/docs**
+
+## Update versioned docs
+
+If you want to update versioned docs, go to [pulsar/site2/website/versioned_docs/](https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs) to find your desired one.
+
+>❗️**BREAKING CHANGE**
+>
+> If you want to update docs for 2.8.x and later versions, follow the steps below.
+
+1. Update the correct doc set.
+   
+    Update these docs ✅ | DO NOT update these docs ❌
+    |---|---
+    [version-2.8.x](https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs/version-2.8.x) | - version-2.8.0-deprecated <br><br> - version-2.8.1-deprecated <br><br> - version-2.8.2-deprecated <br><br> - version-2.8.3-deprecated
+    [version-2.9.x](https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs/version-2.9.x) | - version-2.9.0-deprecated <br><br> - version-2.9.1-deprecated <br><br> - version-2.9.2-deprecated <br><br> - version-2.9.3-deprecated 
+    [version-2.10.x](https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs/version-2.10.x) |- version-2.10.0-deprecated <br><br> - version-2.10.1-deprecated
+
+    For why and how we make this change, see [PIP-190: Simplify documentation release and maintenance strategy](https://github.com/apache/pulsar/issues/16637).
+
+2. Add specific instructions.
+
+    For example, if you want to add docs for an improvement introduced in 2.8.2, you can add the following instructions.
+
+    ```
+    :::note
+
+    This <feature / configuration / xx> is available for 2.8.2 and later versions.
+
+    :::
+    ```
+
+## Update reference docs
+
+If you want to update [Pulsar configuration docs](https://pulsar.apache.org/reference/#/latest/), pay attention to the doc source files.
+
+- Some docs are generated from code **automatically**. If you want to update the docs, you need to update the source code files.
+
+- Some configuration docs are updated **manually** using .md files.
+
+### Update Pulsar configuration docs
```

Comment:
@SignorMercurio could you please help review the lines 99 - 195 from a technical perspective?

Reply:
LGTM